### PR TITLE
Move ClickToGetStarted readme to introduction

### PR DESCRIPTION
**Pull request by orkunkl**
Seeing an empty screen when you enter docs.iov.one webpage is frustrating. I copied https://github.com/iov-one/ClickHereToGetStarted to the introduction and updated it a little bit. We can keep this until we write a better introduction page. Is there anything you want to be added or deleted?

### CODE REVIEW

File: docs/introduction.md [hunk 1]
```diff
@@ -1,7 +1,43 @@
...
+
+We are also building a technology that allows you to **build apps that can access all blockchains**. We are still at the beginning, but you can start building an app that works with Tendermint, Lisk and Rise at the moment, soon ethereum, by using our library **iov-core**. Click [here](https://iov-one.github.io/iov-core-docs/latest/iov-core/index.html "How to Use IOV-Core") to get started.
+
+We have built the first app with iov-core, the **IOV Wallet**, and you can see how iov-core is used and adapted for UI via BCP-Redux. You can use the adaptor from IOV-Core to Redux here: [BCP-redux](https://github.com/iov-one/bcp-redux "BCP-redux repository") and the specific implementation of the [IOV-Wallet](https://github.com/iov-one/iov-wallet "IOV Wallet Repository") (still work in progress, watch it).
```
[the remark this answers is not in the document]
Comment (orkunkl) [reply docs/introduction.md]: Do you mean parts with BCP-redux or all paragraph?

[the remark this answers is not in the document]
Comment (antoineherzog) [reply docs/introduction.md]: Yeah it is oudated!

File: docs/introduction.md [hunk 2]
```diff
@@ -1,7 +1,43 @@
...
+
+You're free to explore the GitHub, all feedback is welcome!
+
+## Introduction to IOV
+
+We're building an interoperability solution for the blockchain industry that will be open source and totally decentralized. Our universal wallet with cross chain atomic swaps and human friendly address formats is already a working prototype. This wallet makes using blockchain tokens as easy for regular consumers as sending and receiving email. Our product suite thus solves user experience problems that we believe must be solved in order to bring blockchain tech into mainstream mass adoption. By design, the product suite also offers mutual benefits to key parties in the blockchain ecosystem (not only end users but also blockchain devs, app creators, entrepreneurs, and network miners) in order to facilitate widespread industry adoption.
```
Comment (antoineherzog) [inline docs/introduction.md]: let's remove as well these 2 sentences:

Our universal wallet with cross chain atomic swaps and human friendly address formats is already a working prototype. This wallet makes using blockchain tokens as easy for regular consumers as sending and receiving email.

File: docs/introduction.md [hunk 3]
```diff
@@ -1,7 +1,43 @@
...
+
+To briefly introduce the product suite:
+
+**IOV Name Service (INS)** is our DNS for blockchains. It is used for name asset registration (including human friendly addresses as well as token name definitions for the chains that implement the BCP.
+
+**IOV Weave** is a framework to quickly build your custom ABCI application to power a blockchain based on the best-of-class BFT Proof-of-stake Tendermint consensus engine. We built IOV Name Service using weave.
+
+**IOV Wallet** is our universal wallet with cross-chain atomic swaps and human friendly addresses. Both desktop and mobile versions are being built.
```
Comment (antoineherzog) [inline docs/introduction.md]: We are not building a mobile version. Just via a chrome extension

File: docs/introduction.md (hunk 2 above)
Comment (orkunkl) [reply docs/introduction.md]: :+1:

File: docs/introduction.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (antoineherzog) [reply docs/introduction.md]: all paragrah!

File: docs/introduction.md [hunk 4]
```diff
@@ -1,7 +1,41 @@
...
+
+You're free to explore the GitHub, all feedback is welcome!
+
+## Introduction to IOV
+
+We're building an interoperability solution for the blockchain industry that will be open source and totally decentralized. Our product suite thus solves user experience problems that we believe must be solved in order to bring blockchain tech into mainstream mass adoption. By design, the product suite also offers mutual benefits to key parties in the blockchain ecosystem (not only end users but also blockchain devs, app creators, entrepreneurs, and network miners) in order to facilitate widespread industry adoption.
+
+To briefly introduce the product suite:
+
+**IOV Name Service (INS)** is our DNS for blockchains. It is used for name asset registration (including human friendly addresses as well as token name definitions for the chains that implement the BCP.
```
Comment (antoineherzog) [inline docs/introduction.md]: I would just say:

 It is used for name asset registration (including human friendly addresses.

remove after --> as well as token name definitions for the chains that implement the BCP.

File: docs/introduction.md [hunk 5]
```diff
@@ -1,7 +1,41 @@
...
+
+We are providing a IOV Name Service which is a register for names used in the blockchain ecosystem, such as personal value addresses, asset names, blockchain names for developers or companies. We believe naming things makes them easier to use for everyone.
+
+We are also providing a toolkit to build apps that can easily use and swap all assets from all blockchains. We believe the future of apps is there.
+
+Enjoy and please send us feedback !
+
+## How to Get Started
+
+We are building technology to **run a blockchain of names**, the IOV Name Service, and build clients for our blockchain. The code to do this is in [Weave](https://github.com/iov-one/weave "Weave Repository"). You can setup a REST API to query IOV Name Service in [BNS-proxy](https://github.com/iov-one/bns-proxy "BNS-proxy repository").You can build a client to the blockchain by using [iov-cli](https://github.com/iov-one/iov-core/blob/master/packages/iov-cli/README.md "IOV-Client Repository").
```
Comment (antoineherzog) [inline docs/introduction.md]: Repo of the IOV Name Service API has changed: https://github.com/iov-one/iovns-rest-api

File: docs/introduction.md [hunk 6]
```diff
@@ -1,7 +1,41 @@
...
+
+You're free to explore the GitHub, all feedback is welcome!
+
+## Introduction to IOV
+
+We're building an interoperability solution for the blockchain industry that will be open source and totally decentralized. Our product suite thus solves user experience problems that we believe must be solved in order to bring blockchain tech into mainstream mass adoption. By design, the product suite also offers mutual benefits to key parties in the blockchain ecosystem (not only end users but also blockchain devs, app creators, entrepreneurs, and network miners) in order to facilitate widespread industry adoption.
+
+To briefly introduce the product suite:
+
+**IOV Name Service (INS)** is our DNS for blockchains. It is used for name asset registration (including human friendly addresses).
```
Comment (davepuchyr) [inline docs/introduction.md]: For North Americans, INS stands for "Immigration & Naturalization Services".  I've had this conversation with @antoineherzog in Slack and he agrees that we should use "IOVNS" instead.

Comment (orkunkl) [reply docs/introduction.md]: ROFL. Yeah we should definitely use IOVNS

File: docs/introduction.md [hunk 7]
```diff
@@ -1,7 +1,41 @@
...
+
+Finally, if you are running a blockchain technology and want **to be compatible with IOV-Core**, we will release full BCP specifications soon. In the meantime, you can start looking at our [code](https://github.com/iov-one/iov-core/tree/master/packages/iov-bns "Codec") and our [documentation](https://iov-one.github.io/iov-core-docs/latest/iov-core/index.html "How to Use IOV-Core") to get familiar with how it all works.
+
+You're free to explore the GitHub, all feedback is welcome!
+
+## Introduction to IOV
+
+We're building an interoperability solution for the blockchain industry that will be open source and totally decentralized. Our product suite thus solves user experience problems that we believe must be solved in order to bring blockchain tech into mainstream mass adoption. By design, the product suite also offers mutual benefits to key parties in the blockchain ecosystem (not only end users but also blockchain devs, app creators, entrepreneurs, and network miners) in order to facilitate widespread industry adoption.
```
Comment (davepuchyr) [inline docs/introduction.md]: I think we should change "network miners" to "block producers" since we have validators, not miners.

Comment (orkunkl) [reply docs/introduction.md]: Agree :+1:

**Pull request merged**